nextflow_schema.json

```
@@ -280,6 +280,24 @@
                 }
             }
         },
+        "generate_samplesheet_options": {
+            "title": "Downstream pipeline samplesheet generation options",
+            "type": "object",
+            "fa_icon": "fas fa-university",
+            "description": "Options for generating input samplesheets for complementary downstream pipelines.",
+            "properties": {
+                "generate_downstream_samplesheets": {
+                    "type": "boolean",
+                    "description": "Turn on generation of samplesheets for downstream pipelines."
+                },
+                "generate_pipeline_samplesheets": {
+                    "type": "string",
+                    "default": "taxprofiler,mag",
+                    "description": "Specify a comma separated string in quotes to specify which pipeline to generate a samplesheet for. Currently supported pipeline(s): taxprofiler,mag",
```

```suggestion
                    "description": "Specify a comma separated string in quotes to specify which pipeline to generate a samplesheet for.",
```

Don't need this if it's in the pattern